components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -866,7 +867,7 @@ TEST_F(PublisherInfoDatabaseTest, GetActivityList) {
   filter_3.excluded = ledger::EXCLUDE_FILTER::FILTER_ALL_EXCEPT_EXCLUDED;
   EXPECT_TRUE(publisher_info_database_->GetActivityList(0,
                                                         0,
-                                                        filter_3,
+                                                        filter_3.Clone(),
```

Comment:
`filter_3.Clone()` should use `std::move(filter_3)` as `filter_3` is no longer used

Reply:
fixed